File: Flix.Backend/Flix.Services/Database/MovieRequestStatus.cs
namespace Flix.Services.Database
{
    /// <summary>
    /// Status of a user-submitted request to add a new movie to the system.
    /// A freshly submitted request is <see cref="Pending"/> (under admin review)
    /// until the administrator approves or rejects it.
    /// </summary>
    public enum MovieRequestStatus
    {
        Pending = 0,
        Approved = 1,
        Rejected = 2
    }
}

File: Flix.UI/flix_desktop/lib/enums/movie_request_status.dart
import 'package:json_annotation/json_annotation.dart';

enum MovieRequestStatus {
  @JsonValue(0)
  pending,
  @JsonValue(1)
  approved,
  @JsonValue(2)
  rejected
}

String getMovieRequestStatus(MovieRequestStatus status) {
  if (status == MovieRequestStatus.pending) {
    return "Pending";
  } else if (status == MovieRequestStatus.approved) {
    return "Approved";
  } else {
    return "Rejected";
  }
}
